saleor/graphql/shop/types.py

```diff
@@ -506,6 +511,10 @@ def resolve_limits(_, _info):
     def resolve_version(_, _info):
         return __version__
 
+    @staticmethod
+    def resolve_schema_version(_, _info):
+        return f'{__version__.split(".")[0]}.{__version__.split(".")[1]}'
```

Comment:
No need to scan and split twice:
```suggestion
    def resolve_schema_version(_, _info):
        major, minor, _ = __version__.split(".", 2)
        return f'{major}.{minor}'
```

Reply:
You are absolutely right, thanks